packages/admin/admin/src/common/Loading.tsx

```diff
@@ -0,0 +1,60 @@
+import { CircularProgress, CircularProgressProps } from "@mui/material";
+import { css, styled } from "@mui/material/styles";
+import React from "react";
+
+export interface LoadingProps extends CircularProgressProps {
+    behavior?: "auto" | "fillParent" | "fillParentAbsolute" | "fillPageHeight";
```

Comment:
How exactly does `auto` work? It seems to me that you're using a different value than the default value in most cases...

Reply:
`auto` is no positioning, it's just there and could be aligned by parent with flex for instance.  
Basically like just using `<CircularProgress />` on it's own.